CHANGELOG.md

```diff
@@ -1,5 +1,76 @@
+    * 1.28.3 or up till 1.29.0
+    * 1.27.7 or up till 1.28.0
+    * 1.26.10 or up till 1.27.0
+    * 1.25.15 or up till 1.26.0
+  * Preview feature registration is no longer required for the KEDA addon
+* Component Updates
+  * Update the aks-app-routing-operator to [version 0.0.7](https://github.com/Azure/aks-app-routing-operator/blob/main/CHANGELOG.md#007---2023-11-04) which includes notable changes in [version 0.0.6](https://github.com/Azure/aks-app-routing-operator/blob/main/CHANGELOG.md#006---2023-10-27).
+    * This update has 3 CVE fixes for the nginx ingress controller.
+      * [CVE-2023-5044](https://nvd.nist.gov/vuln/detail/CVE-2023-5044)
+      * [CVE-2022-4886](https://nvd.nist.gov/vuln/detail/CVE-2023-4886)
+      * [CVE-2023-5043](https://nvd.nist.gov/vuln/detail/CVE-2023-5043)
+    * The following changes are also included:
+      * The AJP protocol is no longer supported.
+      * The `whitelist-source-range` annotation has been renamed to `allowlist-source-range`. Both are currently supported but it is recommended to move to the new annotation `allowlist-source-range`.
+    * The `custom-http-errors` annotation now only supports errors between 400 and 599.
+  * Azure Monitor Metrics [November release](https://github.com/Azure/prometheus-collector/blob/main/RELEASENOTES.md#release-11-03-2023)
+  * KEDA no longer supports addon manager v1.  Charts will move to the v2 addon folder.
+  * Update envoy image version to v1.27.2.  This addresses a bug in the current version that causes the pod to crash with Segment fault.  Additional changes to apiproxy resource limits which addresses resource starvation on large clusters.
+  * Update Azure Disk CSI driver version to v1.29.1 on AKS 1.28, to v1.28.4 on AKS 1.27, to v1.26.7 on AKS 1.26 and 1.25
+  * Update Azure File CSI driver version to v1.29.1 on AKS 1.28, to v1.28.6 on AKS 1.27, to v1.26.9 on AKS 1.26 and 1.25
+  * Update Azure Blob CSI driver version to v1.23.1 on AKS 1.28, to v1.22.3 on AKS 1.27, to v1.21.5 on AKS 1.26 and 1.25
+  * Update cloud-controller-manager image to v1.27.11, v1.26.17, v1.25.22 ([release notes](https://cloud-provider-azure.sigs.k8s.io/blog/))
+    * Fix: deleting a VMMS erroroneously considered a shutdown [4840](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/4804)
+    * Fix: VM name should be obtained from NIC.VirtualMachine.ID instead of NIC name [4878](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/4878)
+    * Fix: public IP address name should be case-insensitive when querying cache [4861](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/4861)
+    * Fix: refresh the load balancer object after updating backend pools to prevent etag mismatch error [4853](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/4853)
+    * Fix: do not tag unmanaged route table [4794](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/4794) 
```

Comment:
What does it fix. Also applicable to the rest, also these are all on the provider release that you should link in the component update.

Reply:
removed - defer to the relnotes for the component